### ISSUE TITLE
Inset parenthesis after function name

### ISSUE THREAD

**Issue by dogovornyachok** [ms-jpq/coq_nvim#472]
Is there any way to automatically insert parenthesis after function name completion?

**Comment by kaiuri** [comment]
yes! just map it using the vim.keymap api as done [here](https://github.com/ms-jpq/coq_nvim/issues/91#issuecomment-1079709069)
